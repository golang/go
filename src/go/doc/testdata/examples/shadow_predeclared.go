// Copyright 2021 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package foo_test

import (
	"fmt"

	"example.com/error"
)

func Print(s string) {
	fmt.Println(s)
}

func Example() {
	Print(error.Hello)
}
