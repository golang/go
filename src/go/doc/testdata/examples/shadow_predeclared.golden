-- .Play --
package main

import (
	"fmt"

	"example.com/error"
)

func Print(s string) {
	fmt.Println(s)
}

func main() {
	Print(error.Hello)
}
